### ISSUE TITLE
Applying group data to Mail object

### ISSUE THREAD

**Issue by dhakan** [elbuo8/sendgrid-django#72]
I've been digging through the source and cannot find a way of sending 
`"asm": {"group_id": 1}` along the request in order to associate a specific unsubscribe link to the user in the mail being sent.

Is this possible? If not, why? And what about all of the other fields in the v3 /mail/send endpoint?

**Comment by jayhale** [comment]
@dhakan, I don't think this is supported by this library at the moment. Could you post an example to test/implement against? I've not attempted unsubscribe groups through this package before, but shouldn't be too challenging to add.

Only a few fields in the API endpoint are implemented right now:
- personalizations
- from
- reply_to (pending PR)
- subject
- content
- attachments


For ref: https://sendgrid.com/docs/API_Reference/Web_API_v3/Mail/index.html

**Comment by dhakan** [comment]
I see, so it looks like my understanding was correct. I'll see what I can do and keep you posted :)

**Comment by danniehakan** [comment]
I'm kind of new to python module distribution. So far I've managed to run `python manage.py install` successfully, though I cannot see which version of django this package is dependent on? I cannot even see that this package depends on django at all.

**Comment by jayhale** [comment]
@dhakan here's an example of an easy way to demonstrate an issue if you can replicate: https://github.com/jayhale/sendgrid-django-61

@danniehakan this package tests against (https://github.com/elbuo8/sendgrid-django/blob/master/.travis.yml):
Django:
- 1.10.4
- 1.9.12
- 1.8.12
Python:
- 2.7
- 3.4
- 3.5
- 3.6
- nightly

It will work with more Django versions, but just isn't tested against all of them.

**Comment by danniehakan** [comment]
Ok, so I've integrated this functionality in my own fork along with proper test coverage. Do I just PR my forked master to this repo, or do I create an issue in the same way you've done here?
